.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,45 @@
+name: golangci-lint
+
+env:
+  SETUP_GO_VERSION: '^1.18'
+
+on:
+  schedule:
+    - cron:  '0 5 * * *'
```

Comment:
Instead of running this on a schedule, can we run this on every change that lands on `master`. Changes don't come over here all that often so it doesn't really need to be run daily. Just when code is pushed.

Reply:
Good point, I have replaced the schedule by on push to master`.`